widgets/BigQueryTable-batchsink.json

```diff
@@ -7,6 +7,14 @@
     {
       "label" : "Service Account and Temporary Google Storage",
       "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName",
+          "widget-attributes" : {
+            "placeholder": "This will be used to uniquely identify this source/sink for lineage, annotating metadata, etc."
```

Comment:
hm for some reason I thought the other plugins had placeholder text here but that doesn't appear to be the case. Sorry, can we remove it so it's consistent.